portfolio/src/main/webapp/comments.html

```diff
@@ -34,6 +34,15 @@
       <option value="25">25</option>
       <option value="-1" selected>All</option>
     </select>
+
+    <label for="lang_code">Select the language:</label>
+
+    <select id="lang_code" onchange="fetchComments()">
+      <option value="en" selected>English</option>
+      <option value="ro">Romanian</option>
+      <option value="fr">French</option>
+      <option value="de">German</option>
```

Comment:
For such a short list it's ok, but generally you would sort the list alphabetically, potentially using the first one or two slots for the majority languages of your user base (and highlighting/separating those to make it clear that they are listed out of order).